src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -378,27 +382,25 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    */
   protected renderContactDetails(person?: IDynamicPerson): TemplateResult {
     person = person || this.personDetails;
+    const userPerson = person as MicrosoftGraph.User;
```

Comment:
Is this necessary?

Reply:
I need it to access the `userPrincipalName` and `businessPhones` properties on the person.

Reply:
Agh, I didn't read far enough :)